hack/get_ci_vm.sh

```diff
@@ -158,10 +158,11 @@ parse_args(){
     VM_IMAGE_NAME="$1"
 
     # Word-splitting is desirable in this case
-    # shellcheck disable=SC2207
+    # shellcheck disable=SC2207,SC2191
```

Comment:
Is this SC2191 necessary after fixing line 164?

Reply:
Yes, this area is more complex than it appears.  Without those disables, we get:

```

In hack/get_ci_vm.sh line 162:
        $(get_env_vars)
        ^-------------^ SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).


In hack/get_ci_vm.sh line 163:
        VM_IMAGE_NAME="$VM_IMAGE_NAME"
        ^------------^ SC2191: The = here is literal. To assign by index, use ( [index]=value ) with no spaces. To keep as literal, quote it.

Did you mean:
        "VM_IMAGE_NAME=""$VM_IMAGE_NAME"


In hack/get_ci_vm.sh line 164:
        UPSTREAM_REMOTE="upstream"
        ^--------------^ SC2191: The = here is literal. To assign by index, use ( [index]=value ) with no spaces. To keep as literal, quote it.

Did you mean:
        "UPSTREAM_REMOTE=""upstream"
```

I'll beef up the comment to explain more fully

Reply:
That's weird. Could it be a difference in the version of shellcheck? I ran this on my laptop before making my comment yesterday:
```console
$ contrib/cirrus/shellcheck.sh
WARNING: It does not appear that containers/automation was installed.
         Functionality of most of this library will be negatively impacted
         This contrib/cirrus/lib.sh was loaded by contrib/cirrus/shellcheck.sh
Shellcheck: PASS
$ rpm -q ShellCheck
ShellCheck-0.7.1-3.fc33.x86_64
```
(I wouldn't have made my comment if shellcheck had failed with SC2191) (the above, needless to say, was run on a tree with the `SC2191` removed from `hack/get_ci_vm.sh` but `SC2207` left in place, because that one is really needed).

Anyhow, it's not worth re-pushing for, but it sounds like we have a problem with ShellCheck, like we always do with linters. It might be worth adding a version dump to the failure message in CI some day.

Reply:
> Could it be a difference in the version of shellcheck?

Oh, that's probably exactly what it is.  In CI we run shellcheck from F33 whereas my laptop is F32.

> but it sounds like we have a problem with ShellCheck, like we always do with linters

The important one is what gets run in CI, but I agree, we gotta take the bad along with the good...at least until the bad exceeds the good :smile: